main.py

```diff
@@ -1,10 +1,57 @@
+import json
+from typing import Union
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race_info: dict[str, list]) -> Race:
+    if Race.objects.filter(name=race_info.get("name")).exists():
+        race = Race.objects.get(name=race_info.get("name"))
```

Comment:
use get_object_or_404() to avoid Error

Reply:
I don't understand what you mean

Reply:
read more [here](https://docs.djangoproject.com/en/4.1/topics/http/shortcuts/#get-object-or-404)

Reply:
![image](https://user-images.githubusercontent.com/32074944/222454676-d49f201e-aedf-4206-b704-218a998a7cc4.png)
I rewrote my code, but the readme advises the option that I used before